packages/redux/src/bags/selectors.ts

```diff
@@ -619,3 +626,88 @@ export const isProductInBag = (
 
   return bagItems.some(bagItem => bagItem.product?.id === Number(productId));
 };
+
+/**
+ * Retrieves a specific bag operation by its id,.
+ *
+ * @param state - Application state.
+ * @param bagOperationId - Unique identifier of the operation.
+ *
+ * @returns - Bag operation entity for the given id.
+ *
+ * @example
+ * ```
+ * import { getBagOperation } from '@farfetch/blackout-redux/bags';
```

Comment:
Always prefer the imports from the root of the package (I have a PR opened which fixes the other files that have the same problem, once it is merged they will be corrected).

```suggestion
 * import { getBagOperation } from '@farfetch/blackout-redux';
```

Reply:
Done!